killrweather-dashboard/app/com/datastax/killrweather/KafkaProperties.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.killrweather
+
+import akka.japi.Util.immutableSeq
+import com.typesafe.config.ConfigFactory
+
+private[killrweather] trait KafkaProperties {
+
+  private val config = ConfigFactory.load
+  protected val KafkaHosts = immutableSeq(config.getStringList("kafka.hosts")).toSet
+  protected val KafkaTopic = config.getString("kafka.topic.raw")
+  protected val KafkaKey = config.getString("kafka.group.id")
+  protected val KafkaBatchSendSize = config.getInt("kafka.batch.send.size")
```

Comment:
Let's not duplicate code, we should refactor ClientHelper with has this functionality already to somewhere the dashboard can use it as well. Will look at where.